src/commands/g.ts

```diff
@@ -0,0 +1,22 @@
+import type { Command } from "./type";
+import type { StoredGifs } from "./gif";
+
+export const g: Command = {
```

Comment:
I think we can give this one a more descriptive name for the command object, even if we are using a shorthand for the actual command